docs/source/extension/extension_dev.rst

```diff
@@ -570,7 +570,56 @@ This ``install.json`` file is used by JupyterLab to help a user know how to mana
 * ``packageName``: This is the package name of the prebuilt extension in the package manager above, which may be different than the package name in ``package.json``.
 * ``uninstallInstructions``: This is a short block of text giving the user instructions for uninstalling the prebuilt extension. For example, it might instruct them to use a system package manager or talk to a system administrator.
 
+.. _trove_classifiers:
+
+PyPI Trove Classifiers
+""""""""""""""""""""""
+
+Extensions distributed as Python packages may declare additional metadata in the form of
+`trove classifiers <https://pypi.org/classifiers>`. These improve the browsing
+experience for users on `PyPI <https://pypi.org/search>`. While including the license,
+development status, Python versions supported, and other topic classifiers are useful
+for many audiences, the following classifiers are specific to Jupyter and JupyterLab.
+Include each relevant classifier (and its parents) to help describe what your package
+provides to prospective users in your ``setup.py``, ``setup.cfg``, or ``pyproject.toml``.
+
+.. code-block::
+
+    Framework :: Jupyter
+    Framework :: Jupyter :: JupyterLab
+    Framework :: Jupyter :: JupyterLab :: 1
+    Framework :: Jupyter :: JupyterLab :: 2
+    Framework :: Jupyter :: JupyterLab :: 3
+    Framework :: Jupyter :: JupyterLab :: 4
+    Framework :: Jupyter :: JupyterLab :: Extensions
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Themes
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Media Types
+
+Include each relevant classifier (and its parents) to help describe what your package
+provides to prospective users in your ``setup.py``, ``setup.cfg``, or ``pyproject.toml``.
```

Comment:
Duplicate of lines 583-584 above.

Reply:
no super-caring about prose at the mo', but thanks.